source/content/guides/multisite/06-search-replace.md

```diff
@@ -18,33 +18,41 @@ editpath: multisite/07-search-replace.md
+WordPress Multisite (WPMS) Search and Replace is available for [Early Access](/oss-support-levels#early-access) participants. Features for platform search and replace on WPMS sites are in active development. Pantheon's development team is rolling out new functionality often while this product is in Early Access. Visit [#wordpress in our community Slack](https://pantheon-community.slack.com/archives/CT8MC5Y0K) (you can sign up for the [Pantheon Slack channel here](https://slackin.pantheon.io/) if you don't already have an account) to learn how you can enroll in our Early Access program. Please review Pantheon's [Software Evaluation Licensing Terms](https://legal.pantheon.io/#contract-hkqlbwpxo) for more information about access to our software.
 
 </Alert>
 
-This section provides information on how to configure automatic platform search and replace for URLs. This feature can be used with both subdirectory and subdomain WordPress Multisites.
+This section provides information on how to configure automatic platform Search and Replace for URLs. This feature can be used with both subdirectory and subdomain WordPress Multisites.
 
-Search and replace runs on the platform as part of creating an environment, deploying to Test or Live, and when cloning content between environments. There is currently a limit of 25 sites that can be configured for search and replace. We expect this number to increase before General Availability.
+Search and Replace runs on the platform as part of creating an environment, deploying to Test or Live, and when cloning content between environments. There is currently a limit of 25 sites that can be configured for search and replace. We expect this number to increase before General Availability.
 
 ## Requirements
 
-You must have the following to enable search and replace:
+You must have the following to enable Search and Replace:
 
 - A [WordPress Multisite](/guides/multisite)
 
-## Enable Search and Replace
+## Search and Replace Configuration
 
-You can enable search and replace between environments on a WPMS site.
+Search and Replace will automatically work for you without any configuration if you have a new `_subdirectory multisite_`. However, you can modify the behavior of Multisite Search and Replace by editing the `search_replace` value.
```

Comment:
@jazzsequence Could we remove this paragraph for this PR? Ideally, we merge the new pantheon.yml changes for EA participants then put this paragraph in a new PR

Reply:
Remove wholesale or remove and revert to the original content?

Reply:
Revert to the original content :)

Reply:
```suggestion
You can enable search and replace between environments on a WPMS site.
```

Reply:
Note for future selves: At some point in the future when we've implemented, we will want to switch this _back_.